src/panels/lovelace/cards/hui-thermostat-card.ts

```diff
@@ -204,13 +204,25 @@ export class HuiThermostatCard extends hassLocalizeLitMixin(LitElement)
       min: stateObj.attributes.min_temp,
       max: stateObj.attributes.max_temp,
       sliderType: _sliderType,
+      step: this._computeTemperatureStepSize(),
```

Comment:
I think we should set this value before the jquery slider set up. and also take step out of the defaults in the top of the file

Reply:
I agree with removing the default, I missed that, but not sure what you mean by setting before the slider setup.

Reply:
Are you just saying to do a
```
const step = this._computeTemperatureStepSize();
```
to pass to the constructor?